docs/source/guide/deployment/planning/services.rst

```diff
@@ -0,0 +1,17 @@
+########
+Services
+########
+
+
+The RSTUF API is a service that exposes a web service. By default, the
+container image exposes port ``80`` or port ``443`` when
+:ref:`using HTTP certificates <guide/repository-service-tuf-api/Docker_README:(Optional) `SECRETS_RSTUF_SSL_CERT\`>`.
+Depending on your RSTUF use case, you might need to expose these ports
+externally.
+
+The RSTUF Worker is not a service but a workload. No port exposed.
```

Comment:
"No port exposed."  Do you mean "No port should be exposed." or "No port is exposed."  What is the meaning of this phrase?

Reply:
No port is exposed (by RSTUF Worker).